src/plugins/auto/executable_network.hpp

```diff
@@ -75,9 +75,9 @@ enum AutoLoadContextIndex {
      CONTEXTNUM = 2
 };
 
+using Time = std::chrono::time_point<std::chrono::steady_clock>;
```

Comment:
this should be used in the cpp file as well

Reply:
sorry I didn't get it, the cpp file should included the header file

Reply:
I meant the std::chrono::steady_clock (whihc is used extensively in the cpp file) is worth the dedicated 'using' as well